packages/fxa-auth-server/lib/oauth/logging/summary.ts

```diff
@@ -0,0 +1,96 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { Request, UserCredentials } from 'hapi';
+import logging from './';
+
+const logger = logging('summary');
+
+interface SummaryRequest extends Request {
+  app: {
```

Comment:
Looks like `Request` already has an `app` property on it.  Might be an issue if some code expects `SummaryRequest.app` to work like `Request.app`.

Reply:
Hm, this might be a spot where I'm taking the wrong approach -- I'm extending `Request.app` to add `remoteAddressChain` because it doesn't appear to exist on it by default. `Request.app` doesn't appear to have any reachable properties from what I can see.

Reply:
Haha `ApplicationState` is just an empty interface.  I think we are good here.  Also, `SummaryRequest` is only used in one spot anyway.